app/Platform/Jobs/UpdateGameMetricsJob.php

```diff
@@ -13,6 +14,7 @@
 
 class UpdateGameMetricsJob implements ShouldQueue, ShouldBeUniqueUntilProcessing
 {
+    use Batchable;
```

Comment:
nit: I think we may want to also include:
```php
if ($this->batch()?->cancelled()) {
    return;
}
```
in `handle()`. Without it, I believe batch cancellation is ignored.

Reply:
done